[2020-04-26 17:22:48,668] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T17:17:40.083377+00:00 [queued]>
[2020-04-26 17:22:48,692] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T17:17:40.083377+00:00 [queued]>
[2020-04-26 17:22:48,692] {{taskinstance.py:866}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 17:22:48,693] {{taskinstance.py:867}} INFO - Starting attempt 2 of 2
[2020-04-26 17:22:48,693] {{taskinstance.py:868}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 17:22:48,713] {{taskinstance.py:887}} INFO - Executing <Task(BashOperator): persiste_dados_mongodb> on 2020-04-26T17:17:40.083377+00:00
[2020-04-26 17:22:48,719] {{standard_task_runner.py:53}} INFO - Started process 26368 to run task
[2020-04-26 17:22:48,787] {{logging_mixin.py:112}} INFO - Running %s on host %s <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T17:17:40.083377+00:00 [running]> 61b2702b37e2.semantix
[2020-04-26 17:22:48,823] {{bash_operator.py:82}} INFO - Tmp dir root location: 
 /tmp
[2020-04-26 17:22:48,826] {{bash_operator.py:105}} INFO - Temporary script location: /tmp/airflowtmpk7yai82j/persiste_dados_mongodbkqdaro6a
[2020-04-26 17:22:48,826] {{bash_operator.py:115}} INFO - Running command: /usr/local/airflow/dags/scripts/requests_database.py --metod POST --company Cielo --collection scores --jsondata {'answered': '6666', 'reclamations': '6151', 'response_time': '15 dias e 21 horas ', 'unanswered': '113'}
[2020-04-26 17:22:48,837] {{bash_operator.py:122}} INFO - Output:
[2020-04-26 17:22:48,876] {{bash_operator.py:126}} INFO -   File "/usr/local/airflow/dags/scripts/requests_database.py", line 50
[2020-04-26 17:22:48,877] {{bash_operator.py:126}} INFO -     if NOT date == '':
[2020-04-26 17:22:48,877] {{bash_operator.py:126}} INFO -               ^
[2020-04-26 17:22:48,878] {{bash_operator.py:126}} INFO - SyntaxError: invalid syntax
[2020-04-26 17:22:48,884] {{bash_operator.py:130}} INFO - Command exited with return code 1
[2020-04-26 17:22:48,896] {{taskinstance.py:1128}} ERROR - Bash command failed
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/bash_operator.py", line 134, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2020-04-26 17:22:48,899] {{taskinstance.py:1170}} INFO - All retries failed; marking task as FAILED.dag_id=ReclameAqui, task_id=persiste_dados_mongodb, execution_date=20200426T171740, start_date=20200426T172248, end_date=20200426T172248
[2020-04-26 17:22:58,653] {{logging_mixin.py:112}} INFO - [2020-04-26 17:22:58,653] {{local_task_job.py:103}} INFO - Task exited with return code 1
